src/Common/ChartLayouts/Standard View.xml

```diff
@@ -2,7 +2,8 @@
     <DataBrowserColumnSettings>
-      <GridColumnSettings columnName="RepositoryName" caption="Repository" visible="1" groupIndex="0" visibleIndex="0" width="0" />
+      <GridColumnSettings columnName="Category" caption="Category" visible="1" groupIndex="1" visibleIndex="0" width="0" />
```

Comment:
I wonder if it should be `groupIndex="1"` or `groupIndex="0"` ?
or is it not relevant (DevExpress just sorts by GroupIndex and the absolute value is not of importance)?

Reply:
@Yuri05 you are right, actually "groupIndex = 0" makes more sense